src/tests/posix/utime_test.c

```diff
@@ -0,0 +1,74 @@
+/**
+ * @file
+ * @brief test for utime
+ *
+ * @author  Filipp Chubukov
+ * @date    21.04.2019
+ */
+
+#include <fcntl.h>
+#include <utime.h>
+#include <sys/stat.h>
+#include <embox/test.h>
+#include <errno.h>
+#include <unistd.h>
+
+#define TEST_EXIST_FILE "test"
+#define TEST_NOT_EXIST_FILE "not_exist_test"
+EMBOX_TEST_SUITE("posix/utime_test");
+
+TEST_CASE("check changed time") {
+	struct utimbuf prev_time;
+	struct utimbuf new_time;
+
+	struct stat filestat;
+	int fd;
+
+	new_time.actime = 999;
+	new_time.modtime = 999;
+
+	test_assert(0 == (fd = creat(TEST_EXIST_FILE, 0666)));
+	test_assert(0 == close(fd));
+
+	test_assert(0 == stat(TEST_EXIST_FILE, &filestat));
+
+	prev_time.actime = filestat.st_ctime;
+	prev_time.modtime = filestat.st_mtime;
+
+	test_assert(utime(TEST_EXIST_FILE, &new_time) == 0);
```

Comment:
Try to use consistent style:

```
assert(0 == f(...))
assert(0 == g(...))
```
or
```
assert(f(...) == 0)
assert(g(...) == 0)
```

but not mix of both.

Reply:
fixed